src/types/dropdown-menu.ts

```diff
@@ -14,3 +14,19 @@ export const AppearanceArray = ['menu', 'selectionMenu'] as const
 export type Appearance = AnyElementOf<typeof AppearanceArray>
 
 export type DropdownItemType = 'link' | 'button' | 'default'
+
+export type DropdownItemRenderedType = 'link' | 'router-link' | 'button' | 'default'
+
+export interface IDropdownItemRenderedComponent {
```

Comment:
nit: you don't need to prefix the interfaces with `I` -- this can just be `DropdownItemRenderedComponent`

Reply:
fixed